examples/alchemy-daapp/.env.example

```diff
@@ -1,8 +1,19 @@
 # When adding additional environment variables, the schema in "/src/env.mjs"
 # should be updated accordingly.
 
-ALCHEMY_API_URL=https://polygon-mumbai.g.alchemy.com/v2/your-api-key
-
-NEXT_PUBLIC_NFT_CONTRACT_ADDRESS=0xYOURADDRESSHERE
-NEXT_PUBLIC_SIMPLE_ACCOUNT_FACTORY_ADDRESS=0xYOURADDRESSHERE
-NEXT_PUBLIC_PAYMASTER_POLICY_ID=some-policy-id
\ No newline at end of file
+POLYGON_ALCHEMY_API_URL=https://polygon-mainnet.g.alchemy.com/v2/apiKey
```

Comment:
In case we want to point to staging?

Reply:
oh is `apiKey` an actual key? I was referring to how you'd actually get an api key into the url at the then, if `apiKey` isn't one.

Reply:
I'm not following sorry haha. This would be where someone would enter their own alchemy RPC URLs or API KEYS.

Reply:
Yeah, I'm curious why you're appending the literal string "/apiKey" to a URL like "https://polygon-mainnet.g.alchemy.com/v2". Are you using it as a n example?

Doesn't everyone need to add their own API Key instead of using the url with "/apiKey"> If so, maybe a comment to clarify would be helpful. Otherwise, people might assume "apiKey" is a working API Key.